client/site-profiler/components/metrics-section/index.tsx

```diff
@@ -54,7 +54,6 @@ const Title = styled.div`
 
 const Subtitle = styled.span`
 	cursor: pointer;
-	color: var( --studio-gray-70 );
 	font-family: 'SF Pro Text';
 	font-size: 16px;
```

Comment:
nit: I would use `1rem` which defaults to `16px` so it would scale with the rest of the site

Reply:
https://github.com/Automattic/wp-calypso/pull/90901/commits/03d9878ff6856f4b66e8a99ae248011e9c83dd94